### PR TITLE
TSK-1098: My issues list

### PR DESCRIPTION
**Pull request by haiodo**
# Contribution checklist

## Brief description
* TSK-1098: My issues list + Space name filter
* Improve scroll speed
* Fix submenu tooltip
image
## Checklist

* [ ] - UI test added to added/changed functionality?
* [x] - Screenshot is added to PR if applicable ?
* [x] - Does the code work? Check whether function and logic are correct.
* [x] - Does Changelog.md is updated with changes?
* [x] - Does the translations are up to date?
* [x] - Does it well tested?
* [x] - Tested for Chrome.
* [x] - Tested for Safari.
* [x] - Go through the changed code looking for typos, TODOs, commented LOCs, debugging pieces of code, etc.
* [x] - Rebase your branch onto master and upstream branch
* [x] - Is there any redundant or duplicate code?
* [x] - Are required links are linked to PR?
* [x] - Does new code is well documented ?

## Related issues

A list of closed updated issues

### CODE REVIEW

File: packages/ui/src/components/EmojiPopup.svelte [hunk 1]
```diff
@@ -1,18 +1,21 @@
 <script lang="ts">
-  import emojiRegex from 'emoji-regex'
-  import { getContext } from 'svelte'
-  import { createEventDispatcher } from 'svelte'
   import { IntlString } from '@hcengineering/platform'
-  import { Label, tooltip, Scroller, emojiSP } from '@hcengineering/ui'
-  import type { AnySvelteComponent } from '@hcengineering/ui'
+  import emojiRegex from 'emoji-regex'
+  import { createEventDispatcher, getContext } from 'svelte'
+  import plugin from '../../../text-editor/src/plugin'
```
Comment (BykhovDenis) [inline packages/ui/src/components/EmojiPopup.svelte]: Please fix import

File: models/tracker/src/index.ts [hunk 2]
```diff
@@ -1020,19 +1020,19 @@ export function createModel (builder: Builder): void {
...
-          // }
+          },
+          {
+            id: 'all-issues',
+            position: 'top',
+            label: tracker.string.AllIssues,
+            icon: tracker.icon.Issues,
+            component: tracker.component.IssuesView,
+            componentProps: {
+              query: { '$lookup.space.archived': false },
```
Comment (BykhovDenis) [inline models/tracker/src/index.ts:1031]: For discuss, maybe we can use spaceSecurityMiddleware to hide archive spaces in future?

**Review by BykhovDenis — APPROVED** [review]

File: packages/ui/src/components/EmojiPopup.svelte (hunk 1 above)
Comment (haiodo) [reply packages/ui/src/components/EmojiPopup.svelte]: Fixed.

File: models/tracker/src/index.ts (hunk 2 above)
Comment (haiodo) [reply models/tracker/src/index.ts:1031]: They still should be available for Vacancies, so better to have view option to hide/show them.

**Pull request merged**